common/uxbox/common/geom/shapes.cljc

```diff
@@ -191,6 +191,17 @@
             (assoc :height value)
             (assoc :width (* value proportion)))))))
 
+(defn resize
+  [shape width height]
+  (us/assert map? shape)
+  (us/assert number? width)
+  (us/assert number? height)
+  (assoc shape
+         :width width
+         :height height
+         :x2 (+ (:x1 shape) width)
+         :y2 (+ (:y1 shape) height)))
```

Comment:
You need to update the selrect `(update :selrect resize width height)`

Reply:
fixed